R/eesapi_url.R

```diff
@@ -0,0 +1,91 @@
+#' eesapi_url
+#'
+#' @param api_version EES API version
+#' @param endpoint Name of endpoint, can be get-summary, get-meta, get-data, query-data
+#' @param dataset_id ID of dataset to be connected to
+#' @param dataset_version Verson of dataset to be connected to
+#'
+#' @return The EES URL for connecting to the API
```

Comment:
Should we be specific that it's a string that's returned, e.g.  `A string containing the URL for connecting to the EES API`?

Reply:
Fair, changed as suggested.